README.md

```diff
@@ -50,6 +54,42 @@ git checkout stable
 cd ..
 ```
 
+## Setup Instructions for Windows 11 with Visual Studio 2022
+
+1. **Install Visual Studio 2022**: Download and install from [Microsoft Visual Studio](https://visualstudio.microsoft.com/vs/). If previously installed, navigate to the "Tools" tab and select "Get tools and features".
+2. During setup, select the "Desktop Development with C++" workload. Ensure the following components are installed:
```

Comment:
Maybe we can make this a drop down list to keep the readme "glance-able". Check out markdown dropdowns examples in our [documentation](https://github.com/EmotiBit/EmotiBit_Docs/blob/master/Keep_emotibit_up_to_date.md#add-adafruit-feather-boards-to-arduino-ide)